geojson_pydantic/geometries.py

```diff
@@ -159,9 +159,9 @@ def check_closure(cls, coordinates: List) -> List:
         return coordinates
 
     @property
-    def exterior(self) -> LinearRing:
+    def exterior(self) -> Union[LinearRing, None]:
         """Return the exterior Linear Ring of the polygon."""
-        return self.coordinates[0]
+        return self.coordinates[0] if self.coordinates else None
```

Comment:
The return type here is probably debatable. Since an exterior should be a LinearRing. But if there is no Polygon, is there any exterior? 
I considered `[]` but:
- `LinearRing` has a min length of 4. The type checker doesn't actually know that, and will accept it, but it felt wrong. 
- There doesn't appear to really be a clean way to say a literal empty list as `Literal[[]]` is not valid. 

Ended up going with None because it seemed the most clear.

Reply:
```python
In [1]: from shapely.geometry import Polygon

In [2]: Polygon()
Out[2]: <POLYGON EMPTY>

In [3]: Polygon().interiors
Out[3]: []

In [4]: Polygon().exterior
Out[4]: <LINEARRING EMPTY>
```

shapely returns `[]` for interiors and return empty Linearring for exterior

Reply:
I did see that but a `LINEARRING EMPTY` would not be valid GeoJSON since it states a `LinearRing` has a min length of 4.  Seemed like `None` would be the closest thing. 

The interiors function we have is a generator but `list(Polygon(...).interiors)` matches the `[]` of shapely.